cluster/src/main/java/com/hartwig/pipeline/calling/somatic/BlacklistFilter.java

```diff
@@ -0,0 +1,26 @@
+package com.hartwig.pipeline.calling.somatic;
+
+import java.util.List;
+
+import com.hartwig.pipeline.calling.command.BcfToolsCommandListBuilder;
+import com.hartwig.pipeline.datatypes.FileTypes;
+import com.hartwig.pipeline.execution.vm.BashCommand;
+import com.hartwig.pipeline.execution.vm.OutputFile;
+import com.hartwig.pipeline.stages.SubStage;
+
+class BlacklistFilter extends SubStage {
+
+    public static final String BLACKLIST_FILTER = "BLACKLIST";
+
+    BlacklistFilter() {
+        super("blacklist.filter", FileTypes.GZIPPED_VCF);
+    }
+
+    @Override
+    public List<BashCommand> bash(final OutputFile input, final OutputFile output) {
+        final String expression = BlacklistBedAnnotation.BLACKLIST_BED_FLAG + "=1 || " + BlacklistVcfAnnotation.BLACKLIST_VCF_FLAG + "=1";
```

Comment:
Nit: `String.format` might make this easier to read.

Reply:
agreed